components/select/__examples__/required.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import IconSettings from '~/components/icon-settings';
+import Select from '~/components/select';
+import SelectContainer from '../private/container';
+import Label from '../private/label';
+
+class Example extends React.Component {
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Select>
```

Comment:
The outputted markup is good, but this should all be one component called Select, since I don't foresee any rearrangement of these such as the label after the `select` element, etc.